arm/Microsoft.Storage/storageAccounts/fileServices/deploy.bicep

```diff
@@ -27,31 +27,33 @@ module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
 
 resource storageAccount 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
   name: storageAccountName
+}
 
-  resource fileServices 'fileServices@2021-04-01' = {
-    name: name
-    properties: {
-      protocolSettings: protocolSettings
-      shareDeleteRetentionPolicy: shareDeleteRetentionPolicy
-    }
+resource fileServices 'Microsoft.Storage/storageAccounts/fileServices@2021-04-01' = {
+  name: name
+  parent: storageAccount
+  properties: {
+    protocolSettings: protocolSettings
+    shareDeleteRetentionPolicy: shareDeleteRetentionPolicy
   }
 }
+
 module fileServices_shares 'shares/deploy.bicep' = [for (share, index) in shares: {
   name: '${deployment().name}-Storage-File-${index}'
```

Comment:
'${uniqueString(deployment().name, location)}-storage-file-${index}' ?

Reply:
Why? :D Did we not agree to just use the unique string on the top-level?